content/upgrade-pop.md

```diff
@@ -242,19 +243,19 @@ sudo apt full-upgrade
 sudo apt install pop-desktop
 ```
 
-### 2. Transisiton to the new release files for Pop!\_OS 21.10
+### 2. Transition to the release files for Pop!_OS 20.04
```

Comment:
We were previously recommending the user upgrade directly to 21.10. Has support seen better success using 20.04 as a stepping stone? (If not, it might make sense to update this section to 22.04 instead of lowering it to 20.04.)

Reply:
Actually, lowering it to 20.04 means this section no longer covers 20.10, which is what it was covering previously.

Upgrading pre-20.04 releases is already covered on lines 158-222.

Reply:
You're right. We've got this doubled up now. I removed the 2nd section. Now all releases older than 20.04 get to 20.04, and the newer releases should just upgrade to 22.04 correct?

Reply:
It could still be useful to have instructions for upgrading 20.10 to 22.04 manually, since we can no longer upgrade pop-upgrade on that release, and I assume there was a reason that section was added.

I noticed that under the remaining `Upgrading older releases` section, the article recommends doing a refresh or clean install for 20.10 upgrades. That would definitely work, but I don't know why that would be recommended while pre-20.04 upgrades still have manual upgrade instructions available.